src/main/java/ch/njol/skript/test/utils/TestOfflinePlayer.java

```diff
@@ -0,0 +1,255 @@
+package ch.njol.skript.test.utils;
+
+import ch.njol.skript.test.runner.TestMode;
+import com.destroystokyo.paper.profile.PlayerProfile;
+import com.destroystokyo.paper.profile.ProfileProperty;
+import io.papermc.paper.persistence.PersistentDataContainerView;
+import org.bukkit.*;
+import org.bukkit.entity.EntityType;
+import org.bukkit.entity.Player;
+import org.jetbrains.annotations.NotNull;
+import org.jspecify.annotations.Nullable;
+
+import java.time.Duration;
+import java.time.Instant;
+import java.util.Date;
+import java.util.Map;
+import java.util.UUID;
+
+public class TestOfflinePlayer implements OfflinePlayer {
+
+	private static final String PLAYER_NAME = "Sahvde";
+	private static final UUID PLAYER_UUID = UUID.fromString("69e37026-2c7d-4255-ac76-9b13a5fe8f74");
```

Comment:
might be best not to use any specific user's information
ideally this can work with a completely made-up profile, but using actual information could obscure cases where we're accidentally relying on looked-up information instead of local info.
Not a big deal though